loops.js

```diff
@@ -51,3 +65,13 @@
 // >> 9
 // ==========================================
 
+for (let i = 0; i < 10; i++) {
+    if (i < 3) {
+        console.log(i)
+    } else if (i > 2 && i < 6) {
+        console.log('> ' + i)
+    } else {
+        console.log('>> ' + i)
+    }
+
```

Comment:
Resultaat is juist. De uitwerking geeft aan dat je ook gebruik kan maken van 3x if statements, zie code:
`  if (i < 3) {
    console.log(i);
  }
  if (i >= 3 && i < 6) {
    console.log('> ' + i);
  }
  if (i >= 6) {
    console.log('>> ' + i);
  }
}`